### ISSUE TITLE
safari 支持PWA

### ISSUE THREAD

**Issue by JOJOforshaun** [mdn/translated-content#17075]
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable

### 与此议题相关的小节或标题？

https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable#%E6%B5%8F%E8%A7%88%E5%99%A8%E6%94%AF%E6%8C%81

### 哪些信息是错误、无用或不完善的？

在桌面上：

    Firefox 和 Safari 不支持在任何桌面操作系统上安装 PWA。
    Chrome 和 Edge 支持在 Linux、Windows、macOS 和 Chromebook 上安装 PWA。

### 你期望看到什么？

在桌面上：

    Firefox 和 Safari 不支持在任何桌面操作系统上安装 PWA。
    Chrome 和 Edge 支持在 Linux、Windows、macOS 和 Chromebook 上安装 PWA。
   Safari 支持在 macOS Sonoma 上安装 PWA。

### 你有什么支持的链接、参考或引用？

https://www.apple.com.cn/macos/sonoma/

### 你还想补充什么内容？

<img width="1062" alt="1700704934037" src="https://github.com/mdn/translated-content/assets/63553362/807a63cc-6524-428f-b8c8-adbde0e2ab5d">

**Comment by yin1999** [comment]
It seems to be a new feature that shipped on macOS Sonoma: https://www.amitmerchant.com/dock-installable-web-apps-in-macos-sonoma/

**Comment by queengooborg** [comment]
Yes, macOS desktop supports installing apps as well in the latest version.  I'm going to transfer this issue to upstream since it's also an English content issue.
